cmd/kops/create_cluster.go

```diff
@@ -1039,3 +1050,22 @@ func parseCloudLabels(s string) (map[string]string, error) {
 	}
 	return m, nil
 }
+
+// setOverrides sets override values in the spec
+func setOverrides(overrides []string, cluster *api.Cluster, instanceGroups []*api.InstanceGroup) error {
+	for _, override := range overrides {
+		kv := strings.SplitN(override, "=", 2)
+		if len(kv) != 2 {
+			return fmt.Errorf("unhandled override: %q", override)
+		}
+
+		// For now we have hard-code the values we want to support; we'll get test coverage and then do this properly...
+		switch kv[0] {
+		case "cluster.spec.nodePortAccess":
+			cluster.Spec.NodePortAccess = append(cluster.Spec.NodePortAccess, kv[1])
+		default:
+			return fmt.Errorf("unhandled override: %q", override)
```

Comment:
I would probably not understand this error.  

How about:

> Only cluster.Spec.NodePortAccess is supported, and %s was set  More support is planned for the future.

Reply:
Right - hopefully this won't be a permanent state of affairs - and I certainly don't intend to hard-code every field :-)